sql/src/test/java/io/crate/planner/fetch/MultiSourceFetchPushDownTest.java

```diff
@@ -90,11 +90,17 @@ public void testPushDownWithOrder() throws Exception {
 
     @Test
     public void testPushDownWithMultiRelationOrder() throws Exception {
-        pushDown("select a, b from t1, t2 order by x - y");
+        pushDown("select a, b from t1, t2, t3 order by x - y + z");
```

Comment:
is this test really testing the MultiSourceFetchPushDown? isn't most logic done at the RelationSplitter now?
if so, please move the test and if some fetch push down related is left over here, modify and rename the test here